tests/homestead/opcodes/test_selfdestruct_contract_balance.py

```diff
@@ -28,7 +28,7 @@
 
 @pytest.mark.compile_yul_with("Paris")  # Shanghai refuses to compile SELFDESTRUCT
```

Comment:
I believe we can remove this now, by updating the yul code to not use the `selfbalance()` opcode.
```suggestion
@pytest.mark.compile_yul_with("Paris")  # Shanghai refuses to compile SELFDESTRUCT
```